<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elpida::Application::BenchmarksView</class>
 <widget class="QWidget" name="BenchmarksView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="glMain">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbAvailableBenchmarks">
     <property name="title">
      <string>Available benchmarks</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lblSelectBenchmark">
        <property name="text">
         <string>Select benchmark:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="twBenchmarks">
        <column>
         <property name="text">
          <string notr="true">Name</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbExecution">
     <property name="title">
      <string>Execution</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1">
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="chkOpenResult">
        <property name="text">
         <string>Open result in browser after upload.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblTimes">
        <property name="text">
         <string>times.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spnTimes">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="chkUpload">
        <property name="text">
         <string>Upload results anonymously to elpida website.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pbRun">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblRun">
        <property name="text">
         <string>Run it </string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
